ragna/source_storages/_embedding.py

```diff
@@ -0,0 +1,16 @@
+from ragna.core import Chunk
+
+from ragna.core._components import Embedding, GenericEmbeddingModel
+
+
+class MiniLML6v2(GenericEmbeddingModel):
+
+    def __init__(self):
+        from sentence_transformers import SentenceTransformer
```

Comment:
See the other components on how you specify optional dependencies, e.g. 

https://github.com/Quansight/ragna/blob/aa2a70263b373baaf40b4c41701c850e40873ad9/ragna/source_storages/_vector_database.py#L51-L64